src/init.cpp

```diff
@@ -1376,7 +1376,7 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     // see Step 2: parameter interactions for more information about these
```

Comment:
unrelated: Not sure what this comment is trying to say. Could remove in this pull?

Reply:
done